### ISSUE TITLE
SubSource Subtitles

### ISSUE THREAD

**Issue by nexusdiscord** [danamag/stremio-addons-list#381]
### Addon Manifest URL

https://f7094476a780-stremio-subsource.baby-beamup.club/manifest.json

### Addon Description

Addon for getting subtitles from Subsource.net

### Language of Content

None

### Choose Labels

- [ ] anime
- [ ] asian drama
- [ ] bollywood
- [ ] cartoons
- [ ] debrid support
- [ ] http streams
- [ ] live tv
- [ ] metadata
- [ ] misc
- [ ] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [X] subtitles
- [ ] torrents
- [ ] tv shows
- [ ] video games

**Comment by tomtomclub** [comment]
Yes working Well thanks for the work ook Nederlands subs super

**Comment by Drmanhattan84** [comment]
Works great for some of the content that previously were not supported by the official subtitle addon. Example: Patrice O'Neal "Elephant in the room".

**Comment by nexusdiscord** [comment]
> Yes working Well thanks for the work ook Nederlands subs super

> Works great for some of the content that previously were not supported by the official subtitle addon. Example: Patrice O'Neal "Elephant in the room".

you should like the first post if you want to like the addon 😁

**Comment by 80085-69420** [comment]
Would it be possible to add exclusion of SDH and HI subs as a new feature?
I only use English subs, and it can take many attempts to find a matching non-HI sub.

**Comment by nexusdiscord** [comment]
> Would it be possible to add exclusion of SDH and HI subs as a new feature? I only use English subs, and it can take many attempts to find a matching non-HI sub.

Hi, We have added the feature to exclude HI subtitles in version 0.0.5. You will need to reinstall the addon. Enjoy!

**Comment by 80085-69420** [comment]
> > Would it be possible to add exclusion of SDH and HI subs as a new feature? I only use English subs, and it can take many attempts to find a matching non-HI sub.
> 
> Hi, We have added the feature to exclude HI subtitles in version 0.0.5. You will need to reinstall the addon. Enjoy!

Thank you so much! I see you added it to SubDL as well. Now these are the perfect sub addons! :)

**Comment by TwilightMercy** [comment]
Addon down?

**Comment by nexusdiscord** [comment]
> Addon down?

no

**Comment by Drmanhattan84** [comment]
Best subtitle provider!

**Comment by TwilightMercy** [comment]
@nexusdiscord Did you remove Hebrew language?

**Comment by nexusdiscord** [comment]
Yeah, I just saw you all mass reporting the stremio addon, and that's really not cool.

**Comment by TwilightMercy** [comment]
> Yeah, I just saw you all mass reporting the stremio addon, and that's really not cool.

I didn't do anything, just noticed and asked

**Comment by TwilightMercy** [comment]
> @nexusdiscord Did you remove Hebrew language?

Why?

**Comment by kraith** [comment]
> > [@nexusdiscord](https://github.com/nexusdiscord) Did you remove Hebrew language?
> 
> Why?

Maybe because he's Indonesian and mostly Muslim 
 ¯\_(ツ)_/¯

**Comment by TwilightMercy** [comment]
Can you bring back Hebrew lang pleaseV

**Comment by SaeidBaesi** [comment]
thanks u for great addone.
for "Mobland" series addone not load subs in Farsi but load it in Arabic. I checked subsurce site for farsi sub there is lots of subtitles in Farsi language but addone not load it.
Addone load Farsi subs without any problems in other Movies and series like "Silo"

**Comment by SaeidBaesi** [comment]
> > thanks u for great addone. for "Mobland" series addone not load subs in Farsi but load it in Arabic. I checked subsurce site for farsi sub there is lots of subtitles in Farsi language but addone not load it. Addone load Farsi subs without any problems in other Movies and series like "Silo"
> 
> I have made the fix, please try again.

thanks. its working.

**Comment by TwilightMercy** [comment]
> Can you bring back Hebrew lang please?

@nexusdiscord

**Comment by nexusdiscord** [comment]
> Can you bring back Hebrew lang please?

I have restored your language

**Comment by TwilightMercy** [comment]
> > Can you bring back Hebrew lang please?
> 
> I have restored your language

Thank you
